### PR TITLE
Add Multi-field textinput for TaskGroup title and description

### PR DESCRIPTION
**Pull request by probablyArth**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,27 +1,20 @@
...
 
 	tea "github.com/charmbracelet/bubbletea"
 	"github.com/maximiliansoerenpollak/todotui/db"
 	"github.com/maximiliansoerenpollak/todotui/models"
-	"github.com/maximiliansoerenpollak/todotui/types"
 )
 
 func main() {
 	db.Write(db.MemData)
-	db.MemData.TaskGroups = append(db.MemData.TaskGroups, types.TaskGroup{GroupTitle: "TaskGroup 1", Tasks: []types.Task{}, GroupDescription: "Description 1"})
```
Comment (MaximilianSoerenPollak) [inline main.go:19]: A bit confused why this db is no longer needed?

**Review by MaximilianSoerenPollak — APPROVED** [review]
Just explain to me quickly why the DB was no longer needed in the main.go file
The line you deleted.

Other than that LGTM. Approved.

File: main.go (hunk 1 above)
Comment (probablyArth) [reply main.go:19]: This line was just adding a hardcoded value to the list of taskgroups, but now we can add it through interface

Comment (MaximilianSoerenPollak) [reply main.go:19]: Ah perfect.

**Pull request merged**